### PR TITLE
make_estimator() works for RobustMultitaskClassifier, RobustMultitaskRegressor, and Sequential

### PR DESCRIPTION
**Pull request by peastman**
A few notes on changes that were involved in this:

For Sequential, the loss is now an argument to the constructor rather than `fit()`.

Due to quirks of the Estimator API, there's no way to determine the shape of the labels.  I therefore modified Input to allow the shape to be unknown.

The output from RobustMultitaskClassifier is now probabilities instead of logits.

### CODE REVIEW

**Comment by rbharath** [comment]
@peastman Sorry for the slow review. These changes look solid! I'm curious to see if we find users starting to use the growing Estimator API to do useful things.

LGTM
